cypress/integration/unit_tests/checkbox_test.js

```diff
@@ -92,35 +101,238 @@ describe('Unit test for toggleLayerOn', () => {
   });
 
   it('check hidden layer, then uncheck before list evaluation', () => {
-    expect(layerArray[2].displayed).to.equals(false);
+    expect(layerArray[2].displayed).to.be.false;
     expect(layerArray[2].data).to.be.undefined;
 
     const emptyList = [];
     let callback = null;
     stubForEmptyList((callb) => callback = callb);
-    toggleLayerOn(mockFirebaseLayers[2]);
+    const promise = toggleLayerOn(mockFirebaseLayers[2]);
+    expect(promise).to.not.be.null;
+    expect(layerArray[2].displayed).to.be.true;
```

Comment:
Why duplicate these checks that are also in 'loads a hidden layer and displays' test and below? If you want to keep them, at least functionalize?

Reply:
Functionalized. Seems useful to have it for line 130, so need to duplicate at least once, so let's go all in :)